mava/adders/reverb/base.py

```diff
@@ -208,58 +261,65 @@ def write_experience_to_tables(  # noqa
                                 extras={},
                             )
                         else:
-                            new_trans = mava_types.Transition(  # type: ignore
+                            # Create a new transition trajectory
+                            new_trajectory = mava_types.Transition(  # type: ignore
```

Comment:
The trajectory can be either a `Step` or a `Transition` depending whether it is sequence based or transition based. So it might make sense to keep it as is? Not sure. What do you think?

Reply:
Ok sure, no that is fine for now.